### DISCUSSION TITLE
How actively maintained is this repo?

### DISCUSSION THREAD

**Discussion by heath-freenome** [plouc/nivo#2258] [General]
Hey, I'm noticing that there has been little in the way of commits to the repo in quite a while and lots of issues and discussions pending responses. I'm worried that this Repo is no longer being actively maintained. @plouc how is it going?

**Comment by john-latham** [comment]
I'm not speaking for anyone but this PR has some relevant comments:

https://github.com/plouc/nivo/pull/2142#issuecomment-1401309459

**Reply by heath-freenome** [reply]
Yeah, I found that and have already responded.

**Comment by plouc** [comment]
To be transparent, not very actively unfortunately, I couldn't find time for side projects lately, I don't want to abandon the project as I think he still has potential, and I invested a lot in it, but I cannot guarantee anything, I would have to find other maintainers to help with the project. Someone helped me in the past, but probably also had some other interests/constraints.

**Reply by heath-freenome** [reply]
@plouc Let me check with my management to see how much help we could provide you. There are definitely lots of little (and bigger) fixes and features we will be needing from some of the components in the library and two engineers who could be tapped to implement them. We would need your help in doing code reviews and releases at minimum. At least until you felt comfortable with handing off some of that burden. Feel free to reach out to me on discord to talk if that would help.
